src/mailbox.rs

```diff
@@ -1,30 +1,57 @@
 use std::sync::Arc;
 
-use crate::chan::{self, BroadcastQueue, Rx, TxStrong, TxWeak};
+use crate::chan::{self, BroadcastQueue, Rx};
 use crate::recv_future::ReceiveFuture;
+use crate::{Address, WeakAddress};
 
+/// A [`Mailbox`] is the counter-part to an [`Address`].
+///
+/// Messages sent into an [`Address`] will be received in an actor's [`Mailbox`].
+/// Think of [`Address`] and [`Mailbox`] as an MPMC channel.
```

Comment:
Also this discovery makes me super happy! I think this should be really easy to grasp for users!